modules/product/product.service.js

```diff
@@ -111,19 +121,41 @@ class ProductsService {
     return new Product(data).save();
   }
 
+  async addModel(data) {
```

Comment:
I think it is better to create for the model separate folder, resolver and service

Reply:
fixed